base/broadcast.jl

```diff
@@ -15,7 +16,7 @@ export broadcast_getindex, broadcast_setindex!
 broadcast(f) = f()
 @inline broadcast(f, x::Number...) = f(x...)
 @inline broadcast{N}(f, t::NTuple{N}, ts::Vararg{NTuple{N}}) = map(f, t, ts...)
-@inline broadcast(f, As::AbstractArray...) = broadcast_t(f, promote_eltype_op(f, As...), As...)
+@inline broadcast(f, As::AbstractArray...) = _broadcast(f, As...)
```

Comment:
Is there still a point in calling this `_broadcast` instead of `broadcast`? Isn't multiple dispatch enough here?